code/game/machinery/iv_drip.dm

```diff
@@ -8,23 +8,36 @@
 	icon_state = "iv_drip"
 	anchored = FALSE
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
+	var/internal_can = FALSE
+	var/start_can = null
+	var/cantype = null
 	var/mob/living/carbon/attached = null
 	var/mode = IV_INJECTING
 	var/obj/item/reagent_containers/beaker = null
 	var/static/list/drip_containers = typecacheof(list(/obj/item/reagent_containers/blood,
 									/obj/item/reagent_containers/food,
 									/obj/item/reagent_containers/glass))
+/obj/machinery/iv_drip/saline
```

Comment:
move this under all the base procs

Reply:
next I'm not sure where you're telling me to move what to, the object to below every single proc? I don't understand what you mean by proc/1 and proc/2 etc and I don't think the saline can has special procs either